src/String/Extra.elm

```diff
@@ -0,0 +1,22 @@
+module String.Extra exposing (isCapitalized)
+
+{-| Some utilities.
+-}
+
+
+{-| Check if the first character of a string is upper case, unicode aware.
+
+Note that `Char.isUpper` is ASCII only, while `Char.isUpper` works outside ASCII.
```

Comment:
This comment sounds misleading to me: The first part sounds like `isUpper` "works" on ASCII only, but after the comma it says that it "works with non-ASCII" characters.
But it might be different for native speakers.

```suggestion
Note that `Char.isUpper` is ASCII only, while `Char.toUpper` also works outside ASCII.
See https://github.com/elm/core/pull/1138
```

Or maybe (but I'm not sure if that is fully correct :sweat_smile:)

```suggestion
Note that `Char.isUpper` returns the correct result only for ASCII characters, even though it can be called with any Unicode character.
See https://github.com/elm/core/pull/1138
```